lib/gap_intelligence/models/header.rb

```diff
@@ -0,0 +1,13 @@
+module GapIntelligence
+  class Header < Record
+    attribute :name
+
+    def initialize(attrs = {})
```

Comment:
Maybe the issue here is that the API is adding the extra fields when they are not relevant to most of the headers. Only spec headers use unit, position, etc.